docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,62 @@
+# Run Antrea agent simulator
+
+This Documentation describes how to run Antrea agent simulator.
+
+## Build the images
+
+1. Build simulator image
+
+   ```bash
+   make build-scale-simulator
+   ```
+
+2. Build the kubemark image
+
+If you alreay have a kubemark image, please ignore this step.
+
+   ```bash
+   cd $KUBERNETES_PATH
+   make WHAT=cmd/kubemark KUBE_BUILD_PLATFORMS=linux/amd64
+   cp ./_output/local/go/bin/linux_amd64/kubemark cluster/images/kubemark
+   cd cluster/images/kubemark
+   docker build -t kubemark:latest .
+   ```
+
+## Create the yaml file
+
+This demo uses 500 simulators, this command will create a yaml file build/yamls/antrea-simulator.yml
```

Comment:
I didn't see it uses 500 simulators, does it?

Reply:
I used to add REPLICA in makefile, but after talking with @weiqiangt , just set the replica as 1 and use script to modify the number. Maybe it is better to remove this description